krabs/krabs/filtering/event_filter.hpp

```diff
@@ -62,6 +73,11 @@ namespace krabs {
         template <typename U>
         void add_on_event_callback(const U &callback);
 
+		unsigned short provider_filter_event_id()const
```

Comment:
nit: indentation is still wrong in this file. please make sure this change uses spaces, not tabs, and that indents are consistent with the rest of the file.

Reply:
Done